app/models/redirect.rb

```diff
@@ -10,6 +10,7 @@ class Redirect
   field :from_path, type: String
   field :to_path, type: String
   field :route_type, type: String
+  field :segments_mode, type: String
```

Comment:
What happens with existing redirects (in the local database) that do not have a `segments_mode` when the are listed or updated. Would we need a migration?